shell/plugins/clean-html-directive.js

```diff
@@ -15,6 +15,18 @@ const ALLOWED_TAGS = [
   'i',
   'em',
   'strong',
+  'h1',
+  'h2',
+  'h3',
+  'h4',
+  'h5',
+  'h6',
+  'table',
+  'thead',
+  'tr',
+  'th',
+  'tbody',
+  'td',
```

Comment:
Why not `blockquote` for example?

Reply:
will add it... This PR needs to be revisited as it's dependent of #9928 , which will lead to changes here.